utility/install_chromedriver.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash 
 
+# This file installs the given CHROME_DRIVER_VERSION to the docker
+# container downloading a stable version to the dockerfile image
+# If no CHROME_DRIVER_VERSION is declared it will take the local systems
+# CHROME VERSIONS.
+
```

Comment:
same comment as I made in install_chrome.sh. I think this needs a bit of rewording. The comment sounds like it's installing whatever is in CHROME_DRIVER_VERSION and a stable version. Plus, it's not installing CHROME_DRIVER_VERSION, it's installing chromedriver, with the version defined by CHROME_DRIVER_VERSION.